www/webapp/package.json

```diff
@@ -31,7 +31,7 @@
     "@vue/cli-service": "^5.0.8",
     "eslint": "^8.31.0",
     "eslint-plugin-vue": "^9.8.0",
-    "sass": "~1.32.13",
+    "sass": "~1.63.3",
```

Comment:
Adds the huge amount of deprecation notifications.

Reply:
Is that a problem? -- Perhaps it makes more sense to first finish #699? The new Vue version hopefully has code that doesn't use much of the deprecated stuff.

Reply:
The deprecation messages are over hundreds and would hide legitimate warnings.
I would recommend to bump it with the migration to Vue3.

Reply:
sounds good, removed